README.md

```diff
@@ -0,0 +1,209 @@
+# vue-ssr-build
```

Comment:
Did you mean to add this at the root?  We already have this in `docs/README.md`...

Reply:
Nope, that was an accident 😨